### DISCUSSION TITLE
Content title shows quoted text, e.g. "I&rsquo;"

### DISCUSSION THREAD

**Discussion by sesponda** [willfaught/paige#61] [Q&A]
Hi,

My `hugo.toml` is:
```
languageCode = 'en-us'
title = "Things I've made"
```

Which renders OK when I browse to the root page of the site. However, when I click on a post, the title is "Things I&rsquo;ve made"
Is this a bug, or I'm using the template/hugo incorrectly? I researched toml syntax but I could not find a solution.

**Comment by sesponda** [comment]
I've fixed this via:

```
[markup.goldmark.extensions.typographer]
            apostrophe = "'"
```

... but it's not probably the right solution.

**Comment by willfaught** [comment]
Thanks for reporting this. Looking into the issue.

**Comment by willfaught** [comment]
This should be fixed in v0.76.0.
